ApiFirst/ApiFirst/RepositoryList.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
+{
+    public class RepositoryList : ICustomerRepository
+    {
+        private readonly List<Customer> List = new();
```

Comment:
```suggestion
        private readonly List<Customer> _customers = new();
```
Usar IList e ajustar nome da lista para `_customers`